ochubryk/add_task.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<HTML>  
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>Add task</TITLE>
+	</HEAD> 	
+	<BODY>		
+		<h1 align="center">Заголовок первого уровня</h1>
+		<h3>Предварительно форматированный текст:</h3>
+<pre>		
+x | o |
+‐‐‐‐‐‐‐‐‐‐‐
+  | х |
+‐‐‐‐‐‐‐‐‐‐‐
+o | x | o
+</pre>
+		<hr>
+		<h3>Тег address</h3>
+		<address> 		
+		Kuprevicha, 1/1, 220141, Minsk, Belarus. Phone: +375.17.2686911
+		</address>
+		<hr>
+		<h3>Семантические и функциональные строчные теги</h3>
+		<p>Пожелания автора:
+		<DEL><font color="red"> Данный текст убрать из монографии</font></DEL> и заменить его на <u><font color="blue">вновь помещённый текст.</font></u></p>
+		<hr>
+		<h3>Работа со строчными и блоковыми цитатами</h3>
+		<p>Цитата короткая:</p>
+		<q><i>И всё­таки она вертится!</i></q>
+		<p>Цитата занимающая несколько предложений:</p>
+		<blockquote>
+			О сколько нам открытий чудных<BR>
+			Готовят просвещенья дух<BR>
+			И опыт, сын ошибок трудных,<BR>
+			И гений, парадоксов друг,<BR>
+			И случай, бог изобретатель...
+		</blockquote>
+		<hr>
+		<h3>Семантические и функциональные строчные теги</h3>
+		<p><STRONG>Тэг &#60STRONG&#62 делает данный фрагмент текста более важным.</STRONG> <em>Тэг &#60em&#62 менее важным.</em>
+		В то время как теги &#60b&#62 и &#60i&#62 делают текст просто <b>полужирным</b> и <i>курсивом</i> соответственно.</p>
+		<hr>
+		<h3>Теги форматирования машинного языка</h3>
+		<p>При выполнении программного кода на языке PERL:</p>
+		<p><code>		
+		#!/usr/local/bin/perl<BR>
+		print "Content‐type: text/html\n\n";<BR>
+		print "&#60H1&#62Hello World&#60/H1&#62\n";</code></p>
+		<p>Программа выведет:</p>
+		<code>
+		Hello, World!</code>
+		<p>Однако в браузере у нас будет отображаться результат выполнения этой программы:</p>
+		<h2>Hello, World!</h2>
+		<hr>
+		<h3>Комментарии и литералы(мнемоники) в HTML</h3>
+<pre>		
+&#60!-- это комментарий --&#62
+		&#60!-- и это также комментарий,
+		занимающий более одной строки --&#62
+</pre>					
+		<hr>
+		<h3>Теги над­ и подстрочных символов</h3>
+		<p align="center"><b>Магазин работает:</b></p>
+		<p align="center">c 08<u><sup>30</sup></u> до 18<u><sup>30</sup></u></p>
+		<p align="center">обед с 13<u><sup>00</sup></u> до 14<u><sup>00</sup></u></p>
+		<p align="center">выходной: <u>суббота</u>,<u>воскресенье</u></p>
+		<hr>
+		<h3>Вот такой текст</h3>
+		<h3><sup>Э</sup><sub>т</sub><sup>о</sup> <sub>п</sub><sup>р</sup><sub>о</sub><sup>с</sup><sub>т</sub><sup>о</sup> <sub>в</sub><sup>е</sup><sub>с</sub><sup>ё</sup><sub>л</sub><sup>ы</sup><sub>й</sub> <sup>т</sup><sub>е</sub><sup>к</sup><sub>с</sub><sup>т</sup></h3>
+		<hr>
+		<h3>Акроним и курсив</h3>
+		<abbr title="Т.... Ф....">ТФ</abbr>: Y(x)=e<sup><i>iπx</i></sup>+log<sub><i>x</i></sub>4
+		<hr>
+		<h3>Тег small внтури абзаца</h3>
+		<p align="right"><small>Создано при помощи блокнота©</small></p>
```

Comment:
Character "copyright" should be written as mnemonic

Reply:
Fixed